lib/shopify_app/scripttags_manager.rb

```diff
@@ -57,5 +60,11 @@ def scripttag_exists?(src)
     def current_scripttags
       @current_scripttags ||= ShopifyAPI::ScriptTag.all.index_by(&:src)
     end
+
+    def build_src(src)
+      src.respond_to?(:call) ? src.call(shop_domain) : src
+    rescue => e
+      raise CreationFailed, "cannot create script src: #{e.message}"
```

Comment:
I think that we should let this method raise the original exception. It's not creation that failed here, it's some logic in the app that sent incorrect types or data. Thinly wrapping the _real_ exception in `CreationFailed` doesn't add much value and might make it more difficult to figure out what _actually_ happened.